templates/item/macros.j2

```diff
@@ -168,58 +168,84 @@ ${{ item.context.name }}: {{ item.context.value }}{% if item.context.scope != 'p
 
 
 {% macro examples(examples_list, id) %}
-  {% for example in examples_list %}
-    {% set group = ['examples', id, loop.index]|join('_') if example.rendered else null %}
-    {% set code = code_block(
-      language=example.type,
-      content=example.code|escape|safe
-      ) %}
-
-    <div data-sassdoc-example="{{ group or 'simple' }}">
-      {% if example.rendered %}
-        {% set input_id = [group, example.type]|join('_') %}
-        {% set output_id = [group, 'output']|join('_') %}
-
-        <div data-sassdoc="example-tabs">
-          {{ utility.tab(
-            id=input_id,
-            group=group,
-            content=example.type
-            ) }}
+  {% set group = ['examples', id]|join('_') %}
+  {% set lang_types = {
+    'njk': 'markup',
+    'html': 'markup',
+    'j2': 'markup',
+    'css': 'style',
+    'scss': 'style',
+    'sass': 'style',
+    'js': 'interaction',
+    'javascript': 'interaction'
+  } %}
+  {% set render_types = {
+    'njk': 'html',
+    'scss': 'css',
+    'sass': 'css'
+  } %}
+
+  <div data-sassdoc="examples">
+    <div data-sassdoc="example-tabs">
+      {% for example in examples_list %}
+        {% if not (example.description|striptags == 'compiled') %}
           {{ utility.tab(
-            id=output_id,
+            id=[group, example.type]|join('_'),
             group=group,
-            content='output',
-            active=true
+            content=lang_types[example.type],
+            active=true if loop.first else false
             ) }}
-        </div>
+        {% endif %}
+      {% endfor %}
+    </div>
 
-        {{ utility.tab_panel(
-          id=input_id,
+    {% for example in examples_list %}
+      {% if not (example.description|striptags == 'compiled') %}
+        {% call utility.tab_panel(
+          id=[group, example.type]|join('_'),
           group=group,
-          content=code|safe,
-          class='input'
-          ) }}
+          active=true if loop.first else false
+          ) %}
+
+          <div data-sassdoc-example="{{ example.type }}">
+            <div data-sassdoc-title="example">
+              {{ example.type }}
+              {% if example.description %}
+                – <em>{{ example.description|striptags }}</em>
```

Comment:
I wonder if we even want any italic in order to avoid loading another font file. Unless we stick to system fonts only, which is something we sorta discussed yesterday. Maybe a span with a class so we can style it differently might work?